addons/website_event_track/views/website_visitor_views.xml

```diff
@@ -21,6 +21,7 @@
                 <button name="%(website_event_track.event_track_action_from_visitor)d"
                     type="action"
                     class="oe_stat_button" icon="fa-ticket"
+                    groups="event.group_event_user"
```

Comment:
https://github.com/odoo/odoo/blob/14.0/addons/website_event_track/security/ir.model.access.csv/#L15-L16

Are you sure about this? `group_event_user` has acces to that? Shouldn't it be `group_event_manager`

Reply:
Indeed, I was mislead by `event_track_wishlisted_count` being `groups='event.group_event_user'`.
<https://github.com/odoo/odoo/blob/14.0/addons/website_event_track/models/website_visitor.py#L22>